components/Experience.tsx

```
@@ -252,12 +232,24 @@ const WorkCard = ({
         </CardHeader>
-        <hr className="mb-6 opacity-40" />
-        <CardContent>
-          {bullets}
+        <hr className="mb-4 opacity-40" />
+        <CardContent className="pb-6">
+          <p className="mb-3">{description}</p>
+          <button
+            onClick={() => setIsOpen((prev) => !prev)}
+            aria-expanded={isOpen}
+            className="text-sm text-[#238493] hover:text-[#2CA5B8] transition-colors cursor-pointer"
+          >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)